boards/posix/native_posix/doc/index.rst

```diff
@@ -621,3 +624,37 @@ development by integrating more seamlessly with the host operating system:
   A backend/"bottom" for Zephyr's CTF tracing subsystem which writes the tracing
   data to a file in the host filesystem.
   More information can be found in :ref:`Common Tracing Format <ctf>`
+
+Host based flash access
+***********************
+
+In case a flash device is present, the file system partitions on the flash
```

Comment:
can you clarify, a 'flash device is present' ?

Reply:
I will rephrase it to:
```In case flash driver support is enabled```